google-cloud-clients/google-cloud-bigtable/src/main/java/com/google/cloud/bigtable/admin/v2/models/ModifyColumnFamiliesRequest.java

```diff
@@ -42,33 +42,18 @@ public static ModifyColumnFamiliesRequest of(String tableId) {
     return new ModifyColumnFamiliesRequest(tableId);
   }
 
-  /**
-   * Configures the tableId to execute the modifications
-   *
-   * @param tableId
-   */
+  /** Configures the tableId to execute the modifications */
   ModifyColumnFamiliesRequest(String tableId) {
     Preconditions.checkNotNull(tableId);
     this.tableId = tableId;
   }
 
-  /**
-   * Configures the name of the new ColumnFamily to be created
-   *
-   * @param familyId
-   * @return
-   */
+  /** Configures the name of the new ColumnFamily to be created */
```

Comment:
Would it make sense to add `{@link ColumnFamily}`? (and the same applies for all of the other comments)

Reply:
Yes, It makes sense to me as well. I have added `{@link ...}`.